python/cucim/src/cucim/skimage/segmentation/_chan_vese.py

```diff
@@ -7,6 +7,8 @@
 from .._shared.utils import _supported_float_type
 from .._vendored import pad
 
+_one = cp.asarray(1.0, dtype=cp.float32)
```

Comment:
Do the functions always use `float32`? Or could this be other types?

Reply:
It can use float64 as well. We could set it based on `float_type` in the chan_vese function instead of doing it globally up here.